Modules/game_menu_ui.py

```diff
@@ -0,0 +1,24 @@
+# File to store data to create the game menu(s)
+import pygame
+import pygame_menu as pm
+

```

Comment:
Needs two new lines between import statements and classes.

Reply:
fixed!